dev-lang/vala/vala-0.56.12.ebuild

```
@@ -27,13 +27,19 @@ DEPEND="${RDEPEND}
 	)
 "
 BDEPEND="
-	dev-libs/libxslt
+	doc? (
+		dev-libs/libxslt
+		sys-apps/help2man
+	)
 	sys-devel/flex
 	virtual/pkgconfig
 	app-alternatives/yacc
 "
 
 src_configure() {
+	# Omit generation of default docs
+	use doc || sed -e 's/[[:blank:]]doc \\//' -i Makefile.am || die
```

```suggestion
	if ! use doc ; then
			sed -i -e 's/[[:blank:]]doc \\//' Makefile.am || die
	fi
```

But:
1. don't do this in `src_configure` - it belongs in `src_prepare`
2. modifying Makefile.am requires eautoreconf (the GNOME eclass has its own way to trigger this) to avoid triggering maintainer mode